ocaml-lsp-server/test/e2e/__tests__/completionItem-resolve.test.ts

```diff
@@ -3,7 +3,7 @@ import * as LanguageServer from "./../src/LanguageServer";
 
 import * as Types from "vscode-languageserver-types";
 
-xdescribe("textDocument/completion", () => {
```

Comment:
Ah, thanks for fixing this up. I followed the example of completion tests and used `xdescribe()` without knowing it's an alias to skip tests 😅

Reply:
No worry haha. I also just discovered it, the name really is confusing. I disabled it again on some ocaml versions because order changes and it kills the tests.